_includes/supp_org.html

```diff
@@ -16,7 +16,9 @@ <h2>Supporting Organisations</h2>
         QuTiP is proud to be working with these organizations:
     </p>
     <div class="d-flex flex-column flex-md-row align-items-center align-items-md-top justify-content-evenly flex-wrap organizations">
-        <a href="https://www.jst.go.jp/moonshot/en/index.html" target="about:blank"><img src="images/moonshot-program-jst.png"></a>
+        <p>
+            <a href="https://www.jst.go.jp/moonshot/en/index.html" target="about:blank"><img src="images/moonshot-program-jst.png"></a>
+        </p>
```

Comment:
You can remove the p tag and just modify the a tag and the img. This should allow the logo to become bigger and take up a whole row. It's a bit of a dirty fix but it works in this specific case.

```
<a href="https://www.jst.go.jp/moonshot/en/index.html" target="about:blank" style="flex-basis: 100%">
    <img class="me-auto ms-auto" style="max-width: 290px" src="images/moonshot-program-jst.png">
</a>
```